### PR TITLE
Sowieso Bundle

### PR DESCRIPTION
**Pull request by Xirdion**
Adding first sowieso bundle

### CODE REVIEW

File: meta/sowieso/contao-image-styling-bundle/de.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+de:
+    title: Lazy Image Styling
+    description: >
+        Dieses Bundle fügt für jedes Bild, das lazy geladen wird, ein inline-Styling hinzu.
+        Durch das Styling wird ein before-Element zum Image-Container hinzugefügt, der den Platz für das Bild reserviert.
+        Dadurch wird beim Nachladen der Bilder der Content der Webseite nicht beeinflusst.
+        
+        Das padding für das before-Element wird dynamisch für alle verwendeten Bildgrößen berechnet und die entsprechenden Media-Queries im CSS angelegt.
+        
+        Neuere Broswer besitzen hierfür das CSS-Attribut "ratio", doch mit dem berechneten Styling kann man auch ältere Browser ansprechen.
```
Comment (fritzmg) [inline meta/sowieso/contao-image-styling-bundle/de.yml:10]:
```suggestion
        Neuere Browser besitzen hierfür das CSS-Attribut "ratio", doch mit dem berechneten Styling kann man auch ältere Browser ansprechen.
```

**Review by fritzmg — CHANGES_REQUESTED** [review]

File: meta/sowieso/contao-image-styling-bundle/de.yml (hunk 1 above)
Comment (Xirdion) [reply meta/sowieso/contao-image-styling-bundle/de.yml]: Thanks for the hint.

**Review by fritzmg — CHANGES_REQUESTED** [review]
You also need to add lazy, inline, before, padding and Queries to the German allow list, since these are English words that you (intentionally) use in your German description.

**Comment by Xirdion** [comment]
Thanks @fritzmg I have added these words to the German allow list.

File: linter/allowlists/de.txt [hunk 2]
```diff
@@ -688,4 +689,9 @@ Anzeigeregeln
...
 Releases
 Stages
-Stage
\ No newline at end of file
+Stage
+lazy
+padding
+inline
+before
+Queries
```
Comment (fritzmg) [inline linter/allowlists/de.txt]: Please sort the words alphabetically (in the rest of the list).

**Review by fritzmg — CHANGES_REQUESTED** [review]

File: linter/allowlists/de.txt (hunk 2 above)
Comment (Xirdion) [reply linter/allowlists/de.txt]: Done. Sorry for the circumstances

**Review by fritzmg — APPROVED** [review]

File: linter/allowlists/de.txt (hunk 2 above)
Comment (fritzmg) [reply linter/allowlists/de.txt]: That's no problem ;)

**Pull request merged**